sentry-android/src/main/java/io/sentry/android/event/helper/AndroidEventBuilderHelper.java

```diff
@@ -147,18 +147,22 @@ public void helpBuildingEvent(EventBuilder eventBuilder) {
     }
 
     private static String[] getProGuardUuids(Context ctx) {
+        if (cachedProGuardUuids != null) {
+            return cachedProGuardUuids;
```

Comment:
It seems like if there are no debug symbols set (very common) we will run this code every time?

Might be better to return an empty array in the `null` or `exception` case and then do nothing with it if it's `length == 0`?